docs/source/sparse.rst

```diff
@@ -1332,6 +1332,15 @@ To manage checking sparse tensor invariants, see:
 
     sparse.check_sparse_tensor_invariants
 
+To enable sparse output support for :func:`~torch.autograd.gradcheck`
+function, decorate the function with
```

Comment:
I don't think that this renders the way you want: https://docs-preview.pytorch.org/pytorch/pytorch/107150/sparse.html#other-functions

Reply:
Fixed, please review.